src/shared/Microsoft.Git.CredentialManager.Tests/Interop/U8StringConverterTests.cs

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+using Microsoft.Git.CredentialManager.Interop;
+using Xunit;
+
+namespace Microsoft.Git.CredentialManager.Tests.Interop
+{
+    public class U8StringConverterTests
+    {
+        /*
+         * Grinning face with squinting eyes and sweat drop
+         *   😅
+         *   Codepoint : U+1F605
+         *   UTF-16    : 0xD83D 0xDE05
+         *   UTF-8     : 0xF0 0x9F 0x98 0x85
+         *
+         * Latin small letter dotless I
+         *   ı
+         *   Codepoint : U+0131
+         *   UTF-16    : 0x0131
+         *   UTF-8     : 0xC4 0xB1
+         *
+         * Greek capital letter omega
+         *   Ω
+         *   Codepoint : U+03A9
+         *   UTF-16    : 0x03A9
+         *   UTF-8     : 0xCE 0xA9
+         *
+         * Snowman without snow
+         *   ⛄
+         *   Codepoint : U+26C4
+         *   UTF-16    : 0x26C4
+         *   UTF-8     : 0xE2 0x9B 0x84
+         */
+
+        // "Unicode 😅 ıs awesome! Ω ⛄";
+        private const string ComplexString = "Unicode \uD83D\uDE05 \u0131s awesome! \u03A9 \u26C4";
```

Comment:
Is there something about .NET that prevents you from putting the emoji literally as part of the string contents?

Reply:
I do appreciate that you have these unit tests!

Reply:
> Is there something about .NET that prevents you from putting the emoji literally as part of the string contents?

Nothing about .NET prevents me, but this is a precaution against _certain_ IDEs, editors, and version control systems from messing up the file encoding. At least this way the tests will be working with stable data.